lib/slack_transformer/html/paragraph.rb

```diff
@@ -0,0 +1,38 @@
+require 'nokogiri'
+
+module SlackTransformer
+  class Html
+    class Paragraph
+      attr_reader :input
+
+      def initialize(input)
+        @input = input
+      end
+
+      def to_slack
+        fragment = Nokogiri::HTML.fragment(input)
+        previous = nil
+        
+        fragment.children.each do |child|
+          if child.name == 'p' 
```

Comment:
Can you make a constant at the top `P_TAG = 'p'`

Then have something like this:
```
tag_type = child.name
if tag_type == P_TAG
    ...
    previous = child.children.empty? ? nil : tag_type
else
    child.replace("\n#{child.to_html}") if previous == P_TAG            
    previous = tag_type
```